src/statistics/Statistics.ts

```diff
@@ -153,15 +153,22 @@ export class Statistics {
         this.registerGauge('os.freeSwapSpaceSize', () => Statistics.EMPTY_STAT_VALUE);
         this.registerGauge('os.maxFileDescriptorCount', () => Statistics.EMPTY_STAT_VALUE);
         this.registerGauge('os.openFileDescriptorCount', () => Statistics.EMPTY_STAT_VALUE);
-        this.registerGauge('os.processCpuTime', () => Statistics.EMPTY_STAT_VALUE);
-        this.registerGauge('os.systemLoadAverage', () => '[' + os.loadavg().toString() + ']');
+        this.registerGauge('os.processCpuTime', () => {
+            // Nodejs 4 does not support this metric. So we do not print an ugly warning for that.
+            if (Util.getNodejsMajorVersion() >= 6) {
+                return process.cpuUsage().user * 1000;
```

Comment:
why are you multiplying this with 1000? Can you make this a variable?

Reply:
process.cpuUsage().user is microseconds and Java client uses nanoseconds for this. I'll add a comment for this.